WordPress/Tests/Theme/NoDeregisterCoreScriptUnitTest.inc

```diff
@@ -0,0 +1,16 @@
+<?php
+/**
+ * All these examples should fail
```

Comment:
It would be good to have some examples where it should not fail so that we can be sure that the check is not over aggressive and fails valid code.